Solutions/AliterConsultingSoD/Watchlists/AliterSODRisks.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "workspaceName": {
+          "type": "string",
+          "metadata": {
+              "description": "Workspace name for Log Analytics where Sentinel is setup"
+          }
+      },
+        "watchlistdescription": {
+            "type": "string",
+            "metadata": {
+                "description": "Risks for SOD Reporting"
+            }
+        }
+  },
+    "resources": [
+        {
+        "name": "[concat(parameters('workspaceName'), '/Microsoft.SecurityInsights/ReferenceTemplate')]",

```

Comment:
Hi @gajackson1963 Can you please change the referenceTemplate value to watchlistname value as well?
Because in both the watchlists value for name property is same so while deployment it is updating the same watchlist instead of creating two watchlists.
Thanks.